src/utils/types.ts

```diff
@@ -0,0 +1,20 @@
+enum AclOperation {
+  All = "ALL",
+  Read = "READ",
+  Write = "WRITE",
+  Create = "CREATE",
+  Delete = "DELETE",
+  Alter = "ALTER",
+  Describe = "DESCRIBE",
+  DescribeConfigs = "DESCRIBE_CONFIGS",
+  AlterConfigs = "ALTER_CONFIGS",
+}
```

Comment:
```suggestion
export type AclOperation =
  | "ALL"
  | "READ"
  | "WRITE"
  | "CREATE"
  | "DELETE"
  | "ALTER"
  | "DESCRIBE"
  | "DESCRIBE_CONFIGS"
  | "ALTER_CONFIGS";
```

`enum` is very badly supported both in TS and JS. A simpler type works better.

Reply:
We need value, not type checking. See below code where we need enum. i think we can't access value from type.
```
export const SolidLabel: React.VFC<SolidLabelProps> = ({ variant }) => {
  const getVariant = (): Variant => {
    switch (variant) {
      case AclResourceType.Group:
        return { labelColor: "green", content: "G" };
      case AclResourceType.Cluster:
        return { labelColor: "grey", content: "KI" };
      case AclResourceType.Topic:
        return { labelColor: "blue", content: "T" };
      case AclResourceType.TransactionalId:
        return { labelColor: "orange", content: "TI" };
      default:
        return;
    }
  };
  ```

Reply:
Sure you can. That type is simply a `string` restricted to just those values.
In the switch you can use the expected string, Typescript will suggest the right ones and all will be typesafe. Eg: `case "GROUP"`